### PR TITLE
Add more logging to diagnose google_issue_tracker filing issues

### PR DESCRIPTION
**Pull request by micahbales**
Replacing https://github.com/google/clusterfuzz/pull/3494

### CODE REVIEW

**Comment by micahbales** [comment]
@oliverchang Please review, merge, and deploy today as you are able. I'll be available this morning (Sydney time) to talk through this PR if that's helpful.

**Review by oliverchang — APPROVED** [review]

File: src/clusterfuzz/_internal/issue_management/issue_filer.py [hunk 1]
```diff
@@ -328,6 +328,8 @@ def file_issue(testcase,
   logs.log(f'Filing new issue for testcase: {testcase.key.id()}.')
 
   policy = issue_tracker_policy.get(issue_tracker.project)
+  logs.log('policy: %s', policy)
+  logs.log('policy.extension_fields: %s', policy.extension_fields)
```
Comment (oliverchang) [inline src/clusterfuzz/_internal/issue_management/issue_filer.py:332]: you need to do % here.

**Review by oliverchang — COMMENTED** [review]
LGTM with just one comment

File: src/clusterfuzz/_internal/issue_management/issue_filer.py (hunk 1 above)
Comment (micahbales) [reply src/clusterfuzz/_internal/issue_management/issue_filer.py]: Thank you for this catch!

**Pull request merged**